CMakeLists.txt

```diff
@@ -251,7 +251,7 @@ endif ()
 if (MINGW)
   check_cxx_compiler_flag("-Wa,-mbig-obj" FMT_HAS_MBIG_OBJ)
   if (${FMT_HAS_MBIG_OBJ})
-    target_compile_options(fmt PUBLIC "-Wa,-mbig-obj")
+    target_compile_options(fmt PRIVATE "-Wa,-mbig-obj")
   endif()
```

Comment:
I suggest removing this block altogether because this is some old workaround that may not even be applicable any more. We can always reintroduce the workaround as an opt-in if it happens again.

Reply:
The issue #1684 is less than 2 years old...

Reply:
Yeah, but it's unclear what purpose `-Wa,-mbig-obj` serves here. The library itself is very small and unlikely to cause linker issues so it might be solving a problem elsewhere.

Reply:
BTW this option was added in #1716. Maybe @ambitslix can shed light why it was needed.